utility/src/main/java/org/oppia/util/crashlytics/CrashlyticsModule.kt

```diff
@@ -0,0 +1,15 @@
+package org.oppia.util.crashlytics
+
+import com.google.firebase.crashlytics.FirebaseCrashlytics
+import dagger.Module
+import dagger.Provides
+import javax.inject.Singleton
+
+@Module
+class CrashlyticsModule {
```

Comment:
As we discussed earlier, rename this to FirebaseModule not crashlytics. Also change the directory name to firebase

Reply:
Done